public/index.php

```diff
@@ -29,9 +31,13 @@
 // Build PHP-DI Container instance
 $container = $containerBuilder->build();
 
+$psr17Factory = new Psr17Factory();
+
+$decoratedResponseFactory = new DecoratedResponseFactory($psr17Factory, $psr17Factory);
```

Comment:
would you be able to explain what these are? i don't know anything about them 😅

Reply:
Yes ofc, probably something easier done in person (well over a call in this case), maybe we could find an hour or so next week while my students are project-ing and I can talk you through this stuff?